README.md

```diff
@@ -171,3 +171,49 @@ The building process is standard. You just need to notice that the `Dockerfile`s
 DOCKER_USER=$USER ./scripts/build-docker-frontend.sh
 DOCKER_USER=$USER ./scripts/build-docker-backend.sh
 ```
+
+## Deployment
+
+This section covers the internal deployment of Substrate Telemetry to our staging and live environments.
+
+### Deployment to staging
+
+Every time new code is merged to `master`, a new version of telemetry will be automatically built and deployed to our staging environment, so there is nothing that you need to do. Roughly what will happen is:
+
+- An image tag will be generated that looks like `$CI_COMMIT_SHORT_SHA-beta`, for example `224b1fae-beta`.
```

Comment:
@arshamparity What is the idea for the `-beta` postfix? Is that something you want to keep long term or was it more of a temporary thing. To us the staging env is useful to manually test new code, but as we do not plan on doing proper releases for `substrate-telemetry` for the time being, there will also not be "beta versions" released. So is the `-beta` postfix still useful?

Reply:
the idea was to build the images first and test them on staging environment with `-beta` postfix, once we are sure the image is working fine, a new tag should be pushed to GitHub, then that version would be used to build the same docker image with the tag provided. we don't need to necessarily do a release, only a new tag would be sufficient to activate the production pipeline. 
With this logic, public users would be informed about `beta` images and they won't use them for their production as they might be broken, but when a new version without `-beta` is available, they can trust that version and deploy it on their production envs.